.github/workflows/upstream-build.yml

```diff
@@ -61,6 +61,15 @@ jobs:
             -DEXTRA_CONF_FILE=$(pwd)/tests/drivers/dma/chan_blen_transfer/boards/siwx917_rb4338a.conf \
             -v --inline-logs
 
+      - name: Build Scatter-Gather DMA test
+        working-directory: zephyr-silabs
+        shell: bash
+        run: |
+          west twister -s drivers.dma.scatter_gather -p siwx917_rb4338a -- \
+            -DTC_OVERLAY_FILE=$(pwd)/tests/drivers/dma/scatter_gather/boards/siwx917_rb4338a.overlay \
+            -DEXTRA_CONF_FILE=$(pwd)/tests/drivers/dma/scatter_gather/boards/siwx917_rb4338a.conf \
```

Comment:
Isn't there a "D" missing in the first line, i.e. `-DDTC_OVERLAY_FILE=...`? However, note the earlier point from @jerome-pouiller regarding `-x` (I'm not familiar with twister options, so double check with the documentation what the right thing is)